src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -22,20 +15,18 @@
  */
 public class Mutect2FilteringEngine {
     private M2FiltersArgumentCollection MTFAC;
+    private final double contamination;
     private final String tumorSample;
     public static final String FILTERING_STATUS_VCF_KEY = "filtering_status";
-    private final List<Hyperparameters> hyperparametersForReadOrientaitonModel;
 
     public Mutect2FilteringEngine(final M2FiltersArgumentCollection MTFAC, final String tumorSample) {
         this.MTFAC = MTFAC;
+        contamination = MTFAC.contaminationTable == null ? 0.0 : ContaminationRecord.readContaminationTable(MTFAC.contaminationTable).get(0).getContamination();
         this.tumorSample = tumorSample;
-        hyperparametersForReadOrientaitonModel = Hyperparameters.readHyperparameters(MTFAC.hyperparameterTable);
-
     }
 
     // very naive M1-style contamination filter -- remove calls with AF less than the contamination fraction
     private void applyContaminationFilter(final M2FiltersArgumentCollection MTFAC, final VariantContext vc, final Collection<String> filters) {
-        final double contamination = MTFAC.contaminationTable == null ? 0.0 : ContaminationRecord.readContaminationTable(MTFAC.contaminationTable).get(0).getContamination();
```

Comment:
Please ignore the changes surrounding contamination - this simply reverts the code back to how it is in master.